.travis.yml

```diff
@@ -18,7 +18,15 @@ before_install:
   - if [[ -n "$RUN386" ]]; then export GOARCH=386; fi
   - if [[ "$TRAVIS_GO_VERSION" = 1.10* && "$GOARCH" != "386" ]]; then ./vet.sh -install || exit 1; fi
 
+install:
+  - go get -u google.golang.org/appengine/...
+  - mkdir /tmp/sdk
+  - curl -o /tmp/sdk.zip "https://storage.googleapis.com/appengine-sdks/featured/go_appengine_sdk_linux_amd64-1.9.64.zip"
+  - unzip -q /tmp/sdk.zip -d /tmp/sdk
+  - export PATH="$PATH:/tmp/sdk/go_appengine"
+
 script:
   - if [[ "$TRAVIS_GO_VERSION" = 1.10* && "$GOARCH" != "386" ]]; then ./vet.sh || exit 1; fi
+  - if [[ "$TRAVIS_GO_VERSION" = 1.9* ]]; then make testappengine || exit 1; fi
```

Comment:
This still doubles the amount of time it takes to run the tests, because it does them in series for 1.9.

Reply:
....instead, let's add this to the "matrix" section above so we get a new instance for running only the appengine tests.